langkit/templates/pkg_introspection_spec_ada.mako

```diff
@@ -68,6 +68,12 @@ package ${ada_lib_name}.Introspection is
    function Derived_Types (Id : Node_Type_Id) return Node_Type_Id_Array;
    --  Return type references for all direct derivations for Id
 
+   function Is_Derived_From (Id, Parent : Node_Type_Id) return Boolean;
+   --  Return whether true if at least one of the following is true:
```

Comment:
This comment made my head hurt. This is the definition of the function but including the recursive definition in the comment is not very helpful IMHO :) Saying it returns true if the type identified by Id is either equal to parent, directly or indirectly derived from parent would seem better

Reply:
Right, sorry, the comment describes the computation more than what it expresses. I’ll fix that.